### ISSUE TITLE
Content should not be using internal.

### ISSUE THREAD

**Issue by moonheart08** [space-wizards/space-station-14#10708]
## Description
tl;dr some systems and components in Content.* are marked as internal. This makes it impossible for downstreams to use those classes in secondary modules.

SS14 has literally no business marking things internal without good reason, and ApcSystem for example should not be internal.

**Comment by wrexbe** [comment]
I agree, internal doesn't make sense if it's not a library like toolbox

**Comment by ike709** [comment]
I tried having a go at this and it looks like it'll cause conflicts with the gamehud refactor, so I'm holding off until that's merged.

**Comment by wrexbe** [comment]
Just do it, I am planning on fixing the merge conflicts on that later today/tomorrow
